### PR TITLE
Echo SFDX commands

### PR DESCRIPTION
**Pull request by cwallsfdc**
- Echo SFDX CLI commands before invoking
- Use HEROKU_APP_NAME for OrgName, if provided (required=true)

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -56,13 +56,15 @@ debug "STACK: $STACK"
...
 
 # Set target org alias
-TARGET_ORG_ALIAS="ra$SOURCE_VERSION"
-TARGET_ORG_NAME="\"Pipeline Review App: $SOURCE_VERSION\""
-
 if [ "$CI" == "true" ]; then
   # CI doesn't include SOURCE_VERSION, so set it to a default value
-  TARGET_ORG_ALIAS="ci$HEROKU_TEST_RUN_COMMIT_VERSION"
-  TARGET_ORG_NAME="\"Pipeline CI: $HEROKU_TEST_RUN_BRANCH $HEROKU_TEST_RUN_COMMIT_VERSION\""
+  TARGET_ORG_ALIAS="ci-$HEROKU_TEST_RUN_COMMIT_VERSION"
```
Comment (wadewegner) [inline bin/compile:61]: @cwallsfdc Did you update `app.json` so that when calling the test it uses the new convention?

File: lib/sfdx.sh [hunk 2]
```diff
@@ -1,21 +1,26 @@
 #!/usr/bin/env bash
 
+invokeCmd() {
```
Comment (wadewegner) [inline lib/sfdx.sh:3]: @cwallsfdc Should we move this into `lib/common.sh` so that we have it defined in one spot?

File: bin/compile (hunk 1 above)
Comment (cwallsfdc) [reply bin/compile:61]: I changed in AppHub.  Sorry about changing - thought a little separation was nice.

```"test-setup": "./vendor/sfdx/release.sh ci-$HEROKU_TEST_RUN_COMMIT_VERSION && ./bin/test-setup.sh",```

File: lib/sfdx.sh (hunk 2 above)
Comment (cwallsfdc) [reply lib/sfdx.sh:3]: Can do, but `sfdx.sh` uses.

Ideally, we move all SFDX invocations to `sfdx.sh` and then move `invokeCmd` there.

File: bin/compile (hunk 1 above)
Comment (wadewegner) [reply bin/compile:61]: That's cool. Now's the time to make the changes!

**Pull request merged**